Data section edits (#1173)
ACCESS-Hive/access-hive.github.io: docs/model_evaluation/data/index.md

        </div>
    </a>
    <!-- Sharing ACCESS Data -->
    <a href="sharing" class="horizontal-card">
        <div class="card-image-container">
            <img src="/assets/external-links.jpg" alt="MED Conda Environment" class="img-cover">
        </div>
        <div class="card-text-container">
            <span class="bold" >Sharing ACCESS Community Data</span>
            <span>
                Options for sharing ACCESS data and the support available from ACCESS-NRI
            </span>
        </div>
    </a>

>     <!-- ACCESS Output Data Specification - change to link to readthedocs-->

Reviewer: ```suggestion
    <!-- ACCESS Output Data Specifications -->
```

>     <a href="https://access-output-data-specifications.readthedocs.io/en/latest/specification/" class="horizontal-card">

Reviewer: Kinda unrelated to this PR, but relevant to data specification:

Who manages the `access-output-data-specifications` RTD? 
Should we keep the localisation? (`../en/..`) If we're not planning to translate in different languages I would remove that.

Also, I would probably link to the general website instead of the specific page, as that's all related to ACCESS Output Data Specifications
```suggestion
    <a href="https://access-output-data-specifications.readthedocs.io/" class="horizontal-card">
```

        <div class="card-image-container">
            <img src="/assets/model_evaluation/model_earth.jpg" alt="A picture of a seismograph recording seismic waves during an earthquake visualises the link to our Observational Data Catalogue. Image credit: Wf Sihardian—EyeEm/Getty Images" title="Image credit: Wf Sihardian—EyeEm/Getty Images" class="img-contain white-background with-padding">
        </div>
        <div class="card-text-container">

>             <span class="bold" >ACCESS Output Data Specification</span>

Reviewer: ```suggestion
            <span class="bold" >ACCESS Output Data Specifications</span>
```

Please double check and correct all instances of "ACCESS Output Data Specification" by adding the "s" in Specification**s** across the Hive Docs.

            <span>

>                 Documentation on the specification for ACCESS model output data.

Reviewer: ```suggestion
                Documentation on the data specifications for data produced by ACCESS models
```
Taken from [the ACCESS](https://access-output-data-specifications.readthedocs.io/en/latest/specification/)

            </span>
        </div>
    </a>
    <a href="/model_evaluation/data/observations" class="horizontal-card">
        <div class="card-image-container">
            <img src="/assets/model_evaluation/model_evaluation_obs_catalog.jpg" alt="A picture of a seismograph recording seismic waves during an earthquake visualises the link to our Observational Data Catalogue. Image credit: Wf Sihardian—EyeEm/Getty Images" title="Image credit: Wf Sihardian—EyeEm/Getty Images" class="img-cover">
        </div>
        <div class="card-text-container">
            <span class="bold" >Observational Data for Evaluation</span>
            <span>

>                 How to find observational data collections for climate model evaluation.

Reviewer: ```suggestion
                How to find observational data collections for climate model evaluation
```
